src/ServiceControlInstaller.Packaging.UnitTests/AuditDeploymentPackageTests.cs

```diff
@@ -39,6 +39,20 @@ public void Should_package_storages_individually()
             }
         }
 
+        [Test]
+        public void Raven5_should_include_raver_server()
+        {
+            var storage = "RavenDb5";
+
+            using (var zip = deploymentPackage.Open())
+            {
+                var persisterFiles = zip.Entries.Where(e => e.FullName.StartsWith("Persisters/") && e.FullName.Contains(storage)).Select(e => e.FullName).ToList();
+                var persisterFolders = persisterFiles.Select(f => Directory.GetParent(f).Name).Distinct();
```

Comment:
One way of fixing this test is by picking only the top-level folders in the `Persistence` folder and ignoring all subfolders.

```suggestion
                var persisterFolders = persisterFiles.Select(f => f.Split('/').Skip(1).Take(1).Single()).Distinct().ToList();
```